### PR TITLE
Sort by address

### PR DESCRIPTION
**Pull request by pducolin**
Using https://koumoul.com/s/geocoder/api-doc to get missing positions, adding positions to cinemas.json to be able to sort them by distance from a position in frontend.

### CODE REVIEW

**Review by kevinmiguet — APPROVED** [review]
Great job !

File: src/utils/positionHelper.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    name:        string;
+    postcode:    string;
+    housenumber: string;
+}
+
+let positions: acPosition[] = require('./cinePositions.json');
+let cinemas: { [id: string]: Cinema } = require('../../cinemas.json');
+
+export async function getPositionForCine(cine: Cinema): Promise<acPosition> {
+    let pos: acPosition = positions.find((_cine: any) => _cine.address === cine.address )
```
Comment (kevinmiguet) [inline src/utils/positionHelper.ts]: might be safer to use the cine ID here, what do you think ?

Comment (pducolin) [reply src/utils/positionHelper.ts]: yes, I should add it in the sync function

File: src/utils/positionHelper.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    if (pos) {
+        return pos
+    }
+    // look for geoloc for Paris cinemas
+    if (!cine.zipCode) {
+        return null
+    }
+    try {
```
Comment (kevinmiguet) [inline src/utils/positionHelper.ts:33]: how about putting this in a dedicated "fetchAddress" function where we would always write the file when fetching a new address ?

Comment (pducolin) [reply src/utils/positionHelper.ts:33]: I moved the sync part to a separate function, will move the async part too. Shall we push the positions file each time we update it?

Comment (kevinmiguet) [reply src/utils/positionHelper.ts:33]: To be honest I'm not familiar with automatic commits with node, so I don't have a strong opinion about this : ) . However I'd say it's only a nice to have. If you think it's a good idea and really want it, feel free to do it !

**Pull request merged**